app/helpers/routing_helper.rb

```diff
@@ -83,6 +83,17 @@ def default_url_options
     end
   end
 
+  def short_account_path account, *more
+    case account.type
+    when 'ActivityMon::Mon'
+      short_mon_path account, *more
+    when 'ActivityMon::Route'
+      short_route_path account, *more
+    when 'ActivityMon::Trainer'
+      short_trainer_path account, *more
+    end
+  end
```

Comment:
Can we just add this to the previous defs so that it applies to all of the paths and not just `short_*_path`?

Reply:
that would make sense